com.unity.ml-agents/Runtime/Policies/BehaviorParameters.cs

```diff
@@ -57,7 +57,7 @@ enum BehaviorType
         /// <summary>
         /// The associated <see cref="BrainParameters"/> for this behavior.
         /// </summary>
-        internal BrainParameters brainParameters
+        public BrainParameters brainParameters
```

Comment:
Is there any point to making this public if BehaviorParameters and BrainParameters are both internal?

Reply:
All public fields on internal classes are internal